net/respondd/src/respondd.c

```diff
@@ -165,6 +183,60 @@ static const struct respondd_provider_info * get_providers(const char *filename)
 	return ret;
 }
 
+bool schedule_push_request(struct request_schedule *s, struct request_task *new_task) {
+	if (s->length >= SCHEDULE_LEN)
+		// schedule is full
+		return false;
+
+	if (!s->list_head || s->list_head->scheduled_time > new_task->scheduled_time) {
```

Comment:
By storing `t->next` in a `struct request_task **`, you could get rid of the if/else. Probably a matter of taste, but I'd like to avoid the additional level of indentation (see [here](https://github.com/freifunk-gluon/packages/pull/143/files#diff-ba51ba086696fdaa98da33f36b0beb50R154))